app/views/users/subscriptions.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="row">
+  <h3 class="brown-text">Your Twitter subscriptions:</h3>
+</div>
+<div class="row">
+  <ul>
+    <% @twitter_subscriptions.each do |subscription| %>
+      <li class="listed-users">
+        <div class= "centered padded-top clipped-flow" >
+          <%= subscription.screen_name %>
+        </div>
+        <div class="centered padded-bottom padded-top">
+          <%= image_tag(subscription.profile_image_uri, class:"centered") %>
+        </div>
+        <div class="centered">
+          <%= button_to "Unsubscribe", unsubscribe_twitter_path, method: :patch, class: 'btn btn-primary', params: { screen_name: subscription.screen_name } %>
+        </div>
+      </li>
+    <% end %>
+  </ul>
+</div>
+<div class="row">
+  <h3 class="brown-text">Your Vimeo subscriptions:</h3>
+</div>
+<div class="row">
+  <ul>
+    <% @vimeo_subscriptions.each do |subscription| %>
+      <li class="listed-users">
+        <div class= "centered padded-top clipped-flow" >
+          <%= subscription.name %>
+        </div>
+        <div class= "centered padded-bottom padded-top">
+          <% if subscription["pictures"].nil? %>
+            <div>
+            <%= image_tag("default_photo.png", class:"centered resized") %>
+            </div>
+          <% else %>
+            <%= image_tag(subscription["pictures"]["sizes"][1]["link"], class:"centered resized") %>
+          <% end %>
+        </div>
+        <div class= "centered">
+          <%= button_to "Unsubscribe", unsubscribe_vimeo_path, method: :patch, class: 'btn btn-primary', params: { uri: subscription.uri } %>
+        </div>
+      </li>
+    <% end %>
```

Comment:
Looks like you have quite a bit of duplicated code here. I think there might be a couple ways you could refactor this: either create a view helper file that you pass in the subscriptions and/or I think a better approach might be to have one table in your db for twitter users and vimeo users (you could have a column for "provider" where you specify if it's a twitter user or vimeo user)...that way you don't have to do this each loop twice, rather you can just pull all your subscribers from one table.